internals/_init.php

```
@@ -0,0 +1,271 @@
+    
+    $if_modified_since = $_SERVER['HTTP_IF_MODIFIED_SINCE'] ?? null;
+    
+    if ($if_modified_since !== null) {
+        $if_modified_since_time = strtotime($if_modified_since);
+        if ($if_modified_since_time === false) {
+            return;
+        }
+        
+        if ($if_modified_since_time >= $last_modified_time) {
+            http_response_code(304);
+            exit;
```

PHPMD (2.15.0)

[error] The function process_last_modified() contains an exit expression. (undefined)

(ExitExpression)